src/resolvers/intermine/gene-family.ts

```diff
@@ -48,5 +51,10 @@ ResolverMap => ({
                 // @ts-ignore: implicit type any error
                 .then(({data: results}) => results);
         },
+        linkouts: async (geneFamily, _, { dataSources }) => {
+            const {identifier} = geneFamily;
+            console.log("gonna call it with identifier");
```

Comment:
Remove this!

Reply:
eagle eyes!
If you like, we can wait to approve until I've deployed the new linkouts so as to do the proper testing

Reply:
OK, the new microservices deployment is complete, so the new services should return data for gene families with prefix legfed_v1_0 and pangenes with prefix Glycine.pan4 (more to be added, but that should suffice for the rigors of testing)